WalletWasabi.Gui/Dialogs/CannotCloseDialogViewModel.cs

```diff
@@ -164,7 +164,7 @@ private async Task StartDequeueAsync(CancellationToken token)
 							}
 						}
 
-						if (latestException is null) //no exceptions were thrown during the for-each so we are done with dequeuing
+						if (latestException is null) //no exceptions were thrown during the for-each so we are done with Dequeueing
```

Comment:
NACK, same reasoning for the spelling, but also for the capitalization, makes no sense to capitalize.